content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -49,15 +49,10 @@ The output is similar to this:
 job.batch/pi created
 ```
 
-Check on the status of the Job with `kubectl`:
+Check on the status of the Job with `kubectl` and output is similar to this:
```

Comment:
No need to replace, the original text is fine.

Reply:
Reverted. Thanks!!